lib/services/types.proto

```diff
@@ -672,6 +672,24 @@ message AccessRequestFilter {
     RequestState State = 3 [ (gogoproto.jsontag) = "state" ];
 }
 
+// AccessCapabilities is a summary of capabilities that a user
```

Comment:
I think we should make this a full versioned resource. I know we're not persisting this to the backend, but if we change the format of `services.AccessCapabilities` that will make it hard to detect from the client.